vispy/visuals/mesh.py

```diff
@@ -123,10 +90,15 @@ class MeshVisual(Visual):
     >>> mesh = MeshVisual(vertices=vertices, faces=faces)
     """
 
+    _shaders = {
+        'vertex': vertex_shader,
+        'fragment': fragment_shader,
+    }
+
```

Comment:
is this much more accessible than from `Visual.shared_program`?

Reply:
I think of this as more subclass friendly that `shared_program`...maybe. I don't think `shared_program` lets you easily access the shaders as strings.

This being a dictionary is nice in my opinion because it allows for other shaders (ex. geometry) for other Visuals. Maybe this is closer to what a "standard" for all Visuals would look like and that's why I find it appealing. I could be convinced otherwise though.

Reply:
Exactly, `shared_program` is basically only good for setting/getting specific values. I used it before to get the shader, and it was basically impossible :P

Yes, I made it a dict specifically because I thought the geometry one could fit and not be required. I guess we could even move it to the init in the `Visual` base class (it automatically picks from this dict), but maybe that's for another time.

Reply:
you've convinced me! 🙂